Assets/AppCenter/Plugins/iOS/Crashes/CrashesUnity.mm

```diff
@@ -71,3 +73,19 @@ void appcenter_unity_start_crashes()
 {
     [MSAppCenter startService:MSCrashes.class];
 }
+
+void* app_center_unity_crashes_get_error_attachment_log_text(char* text, char* fileName)
+{
+    return (void *)CFBridgingRetain([[MSErrorAttachmentLog alloc] initWithFilename:appcenter_unity_cstr_to_ns_string(fileName) attachmentText:appcenter_unity_cstr_to_ns_string(text)]);
```

Comment:
Please split the method call on multiple lines

Reply:
Ok